extras/parse_travis_log.py

```diff
@@ -177,6 +180,35 @@ def process_static_analysis(f, line):
         if line.startswith(' - clang-format for '):
             d.update(process_clang_format(f, filename))
 
+def process_pep8(f, line):
+    """
+    Process PEP8 output for a certain file.
+    """
+    # Check PEP8 on file pynest/nest/lib/hl_api_connections.py:
+    filename = line.split(' ')[-1].strip()[0:-1]
+    d = set()
+    res = {filename: d}
+    while True:
```

Comment:
I go through the file only once and `f` stores the "pointer" into where we are at reading the file. When i use `iter(f)`, each function has its own state there, correct? Then I have to rewrite the script, which might be not a bad idea, as it will make the individual parsing functions simpler. But I think this would be a topic for another PR.